source/utilities.tex

```diff
@@ -2920,6 +2920,18 @@
 
 \rSec3[optional.ctor]{Constructors}
 
+\pnum
+The exposition-only variable template \exposid{converts-from-any-cvref}
+is used by some constructors for \tcode{optional}.
+\begin{codeblock}
+template<class T, class W>
+inline constexpr bool @\exposid{converts-from-any-cvref}@  // \expos
```

Comment:
We don't need "inline" for variable templates; please drop.

Reply:
See #4625

Reply:
Done